File: P77884.py
def get_divider_count(n):
    
    result = 0
    
    for i in range(1, n+1):
        if not n%i:
            result +=1
    
    return result

def solution(left, right):
    answer = 0
    for i in range(left, right+1):
        if get_divider_count(i) % 2:
            answer -= i
        else:
            answer += i
            
    return answer

File: P77884.java
public class P77884 {
    public int solution(int left, int right) {
        int answer = 0;
        for (int i=left; i<=right; i++) {
            if (countDivider(i)) answer += i;
            else answer -= i;
        }
        return answer;
    }
    public boolean countDivider(int n) {
        boolean result = true;
        for (int i=1; i<=n; i++) if (n%i == 0) result = !result;
        return result;
    }   
}